infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/StatusLogger.java

```diff
@@ -71,6 +71,23 @@ public void warnBellatrixParameterChanged(final String parameterName, final Stri
             Color.YELLOW));
   }
 
+  public void warnMissingProposerDefaultFeeRecipientWithRestAPIEnabled() {
+    log.warn(
+        print(
+            "No default proposer fee recipient has been specified and rest API are enabled. "
+                + "If a Validator Client is going to use this node is strongly recommended to "
+                + "configure it to avoid possible loss of fees gained proposing a block",
```

Comment:
```suggestion
            "No default proposer fee recipient has been specified and rest API is enabled. "
                + "If a Validator Client is going to use this node it is strongly recommended to "
                + "configure it to avoid possible loss of fees gained proposing a block",
```